src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -89,13 +99,34 @@ public function storeOperatesOnASingleChannel()
 
     /**
      * @Given /^the store operates on (?:a|another) channel named "([^"]+)"$/
+     * @Given the store operates on a channel identified by :code code
      */
-    public function theStoreOperatesOnAChannelNamed($channelName)
+    public function theStoreOperatesOnAChannelNamed($channelIdentifier)
     {
-        $channel = $this->channelFactory->createNamed($channelName);
-        $channel->setCode($channelName);
+        $channel = $this->channelFactory->createNamed($channelIdentifier);
+        $channel->setCode($channelIdentifier);
 
         $this->channelRepository->add($channel);
         $this->sharedStorage->set('channel', $channel);
     }
+
+    /**
+     * @Given the channel :channel is enabled
+     */
+    public function theChannelIsEnabled(ChannelInterface $channel)
+    {
+        $channel->enable();
+        $this->channelManager->flush();
+        $this->sharedStorage->set('channel', $channel);
+    }
+
+    /**
+     * @Given the channel :channel is disabled
```

Comment:
I'm 100% against magic :D "Cutting a letter to call a function" smells really bad for me. I think that we have ``setEnabled`` function to use it in situations like this one. We have parameter, that can be true or false (we could even write transformer for such a case :D) and it can be used to set resource ``enabled`` state. If you're still not convinced, I'm for extracting common logic ;)